openbb_terminal/core/sdk/trail_map.csv

```diff
@@ -458,22 +457,20 @@ stocks.fa.fraud,stocks_fa_av_model.get_fraud_ratios,
 stocks.fa.growth,stocks_fa_fmp_model.get_financial_growth,
 stocks.fa.historical_5,stocks_fa_dcf_model.get_historical_5,
 stocks.fa.income,stocks_fa_sdk_helpers.get_income_statement,
-stocks.fa.info,stocks_fa_yahoo_finance_model.get_info,
 stocks.fa.key,stocks_fa_av_model.get_key_metrics,
 stocks.fa.metrics,stocks_fa_fmp_model.get_key_metrics,
 stocks.fa.mgmt,stocks_fa_business_insider_model.get_management,
 stocks.fa.mktcap,stocks_fa_yahoo_finance_model.get_mktcap,stocks_fa_yahoo_finance_view.display_mktcap
 stocks.fa.news,stocks_fa_finviz_model.get_news,
-stocks.fa.overview,stocks_fa_av_model.get_overview,
-stocks.fa.profile,stocks_fa_fmp_model.get_profile,
+stocks.fa.overview,stocks_fa_sdk_helpers.get_overview,
 stocks.fa.pt,stocks_fa_business_insider_model.get_price_target_from_analysts,stocks_fa_business_insider_view.price_target_from_analysts
 stocks.fa.rating,stocks_fa_fmp_model.get_rating,
 stocks.fa.ratios,stocks_fa_fmp_model.get_key_ratios,
 stocks.fa.revfc,stocks_fa_seeking_alpha_model.get_estimates_rev,
 stocks.fa.rot,stocks_fa_finnhub_model.get_rating_over_time,stocks_fa_finnhub_view.rating_over_time
 stocks.fa.score,stocks_fa_fmp_model.get_score,
 stocks.fa.sec,stocks_fa_marketwatch_model.get_sec_filings,
-stocks.fa.sec_fmp,stocks_fa_fmp_model.get_filings,stocks_fa_fmp_view.display_filings
+stocks.fa.sec_fmp,stocks_fa_fmp_model.get_filings,
```

Comment:
For consistency, the commands should be merged into one with a source option

Reply:
which commands in specific?